operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -405,7 +406,15 @@ private void processServerPodWatch(V1Pod pod, String watchType) {
         info.setServerPodBeingDeleted(serverName, Boolean.FALSE);
         // fall through
       case MODIFIED:
-        info.setServerPodFromEvent(serverName, pod);
+        boolean podAlreadyEvicted = info.setServerPodFromEvent(serverName, pod, PodHelper::isEvicted);
+        if (PodHelper.isEvicted(pod) && !podAlreadyEvicted) {
```

Comment:
What's the difference between PodHelper.isEvicted()  vs podAlreadyEvicted ?   When will podAlreadyEvicted be true?

Reply:
PodHelper.isEvicted(pod) is true if the new pod from the PodWatch event is evicted.
podAlreadyEvicted is true if the original pod from getSko(serverName).getPod() was already evicted.

Reply:
renamed to podPreviouslyEvicted. Thanks